UndertaleModLib/Decompiler/Disassembler.cs

```diff
@@ -34,18 +34,24 @@ public static string GenerateLocalVarDefinitions(this UndertaleCode code, IList<
 
             return sb.ToString();
         }
-
+        
         public static string Disassemble(this UndertaleCode code, IList<UndertaleVariable> vars, UndertaleCodeLocals locals)
         {
-            StringBuilder sb = new StringBuilder();
+            // This stringbuilder is shared with the code instruction's ToString methods.
+            // Experimentation has shown that 200 is a good enough starting value for it. 
+            // 300 seemed too high and 100 too low. This may change in the future.
+            StringBuilder sb = new StringBuilder(200);
             if (locals == null && !code.WeirdLocalFlag)
                 sb.Append("; WARNING: Missing code locals, possibly due to unsupported bytecode version or a brand new code entry.\n");
             else
                 sb.Append(code.GenerateLocalVarDefinitions(vars, locals));
 
-            Dictionary<uint, string> fragments = new Dictionary<uint, string>();
+            Dictionary<uint, string> fragments = new();
```

Comment:
You can improve performance by setting an initial capacity:
```suggestion
            Dictionary<uint, string> fragments = new(code.ChildEntries.Count);
```

Reply:
Ah cool, wasn't aware of that